lib/cosmos/tools/cmd_tlm_server/api.rb

```diff
@@ -794,12 +794,14 @@ def get_limits_groups
 
     # (see Cosmos::Limits#enable_group)
     def enable_limits_group(group_name)
+      Logger.info("Enabling Limits Group: #{group_name.upcase}")
```

Comment:
What do you think about putting the log message right in the API. I noticed we did that for enable_limits_group but not for a bunch of other things.